internal/log/formatter/prefix_style.go

```diff
@@ -0,0 +1,52 @@
+package formatter
+
+import (
+	"github.com/puzpuzpuz/xsync/v3"
+)
+
+var (
+	// defeultPrefixStyles contains ANSI color codes that are assigned sequentially to each unique prefix in a rotating order
+	// https://user-images.githubusercontent.com/995050/47952855-ecb12480-df75-11e8-89d4-ac26c50e80b9.png
+	// https://www.hackitu.de/termcolor256/
+	defeultPrefixStyles = []ColorStyle{
+		"66", "67", "95", "96", "102", "103", "108", "109", "139", "138", "144", "145",
```

Comment:
NIT: It might be better to have constants that indicate what these ANSI color codes represent.

e.g.

```go
defaultPrefixStyles = []ColorStyle{
  red, blue, green, ...
}
```

Also note that the variable name is mispelled.

Reply:
Don't make any changes now, but all I was saying is that it would be good to have a constant assigned the value of `"103"` so that I didn't have to search for it in the ANSI color thingy to know what color that was. If we want to change any of these, we'll have to do some looking up to find them.